test/test_exadialect_turbodbc.py

```diff
@@ -4,34 +4,36 @@
 from sqlalchemy.testing import fixtures
 from sqlalchemy.testing import eq_
 
-from sqlalchemy_exasol.turbodbc import (EXADialect_turbodbc, DEFAULT_CONNECTION_PARAMS)
+from sqlalchemy_exasol.turbodbc import EXADialect_turbodbc, DEFAULT_CONNECTION_PARAMS
 
 
 @pytest.mark.skipif(
     "turbodbc" not in testing.db.dialect.driver,
-    reason="This tests are only relevant for the dialect Exasol turbodbc dialect"
+    reason="This tests are only relevant for the dialect Exasol turbodbc dialect",
 )
 class EXADialectTurbodbcTest(fixtures.TestBase):
 
     dialect = None
-    default_host_args = {
-        'exahost': '192.168.1.2..8:1234',
-        'exaschema': 'my_schema'
-    }
+    default_host_args = {"exahost": "192.168.1.2..8:1234", "exaschema": "my_schema"}
     default_user_args = {
-        'uid': 'scott',
-        'pwd': 'tiger',
+        "uid": "scott",
+        "pwd": "tiger",
     }
 
     @staticmethod
     def _assert_connect_args(result, expected, expected_turbodbc):
-        eq_({k: v for k,v in result.items() if k != 'turbodbc_options'}, expected)
-        assert (result['turbodbc_options'].read_buffer_size.megabytes
-                == expected_turbodbc.get('read_buffer_size', 50))
-        assert (result['turbodbc_options'].parameter_sets_to_buffer
-                == expected_turbodbc.get('parameter_sets_to_buffer', 1000))
-        assert (result['turbodbc_options'].use_async_io
-                == expected_turbodbc.get('use_async_io', False))
+        eq_({k: v for k, v in result.items() if k != "turbodbc_options"}, expected)
+        assert result[
+            "turbodbc_options"
+        ].read_buffer_size.megabytes == expected_turbodbc.get("read_buffer_size", 50)
+        assert result[
+            "turbodbc_options"
+        ].parameter_sets_to_buffer == expected_turbodbc.get(
+            "parameter_sets_to_buffer", 1000
+        )
+        assert result["turbodbc_options"].use_async_io == expected_turbodbc.get(
+            "use_async_io", False
+        )
```

Comment:
oh, these look weird, I think we should maybe write them differently from the beginning

Reply:
agreed